airflow/serialization/pydantic/taskinstance.py

```diff
@@ -340,7 +378,6 @@ def get_email_subject_content(
 
         return _get_email_subject_content(task_instance=self, exception=exception, task=task)
 
-    @provide_session
```

Comment:
we shouldn't be providing sessions on the pydantic model -- we shouldn't be using sessions at all.